haskell/haskell.bzl

```diff
@@ -252,6 +252,9 @@ for `haskell_binary`.
 haskell_import = rule(
     _haskell_import_impl,
     attrs = dict(
+        deps = attr.label_list(
+            doc = "List of other Haskell libraries to be linked to this target.",
```

Comment:
Not just Haskell libraries, right? E.g. `gmp` is a `cc_library`.

`haskell_import` doesn't really build anything. So, having `deps` for it might seem a bit obscure. I think the docs should explicitly explain that the purpose of this is to forward dependencies such as `libgmp`.